contributors.json

```diff
@@ -1,4 +1,12 @@
 {
+    "doug":{
+        "name":"Douglas W. Oldenburg",
+        "affiliation":"University of British Columbia",
+        "location":"Vancouver, BC",
+        "email":"doug@eos.ubc.ca",
+        "url":"http://gif.eos.ubc.ca/",
+        "avatar":"http://www.simpeg.xyz/s/img/people/doug.png",
```

Comment:
the comma at the end of this line is causing a fail (json always expects another entry after a comma)